src/foam/mlang/mlang.js

```diff
@@ -893,7 +893,6 @@ return stmt.toString();`
         `java.util.List<Predicate> args = new java.util.ArrayList<>();
 boolean update = false;
 True TRUE = new True();
-False FALSE = new False();
```

Comment:
I think we can remove the previous line also, since the TRUE constant is already defined on line 3344.

Reply:
but, this will be generated in Or.java that don't access to at that scoop.